Symfony/CS/Tests/Fixer/Contrib/HeredocToNowdocFixerTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Gregor Harlan <gharlan@web.de>
+ *
+ * @internal
+ */
+final class HeredocToNowdocFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideTestFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideTestFixCases()
+    {
+        return array(
+            array(<<<'EOF'
+<?php $a = <<<'TEST'
+TEST;
```

Comment:
this test is useless. It is already covered by the later item (as ``makeTest`` always ensure that applying the fixer on the expected output triggers no additional change)